sdk/core/core/test/test_az_context.c

```diff
@@ -0,0 +1,33 @@
+#include <az_test.h>
+
+#include <az_context.h>
+#include <az_result.h>
+
+#include <stddef.h>
+
+#include <_az_cfg.h>
+
+void test_az_context()
+{
+  void *key = "k", *value = "v";
```

Comment:
nit: I'd declare each on a separate line, and as `void* const`

Reply:
We can't do that for 'value' here because of the call to get_value below.